spring-cloud-gcp-autoconfigure/src/main/java/com/google/cloud/spring/autoconfigure/sql/CloudSqlEnvironmentPostProcessor.java

```diff
@@ -86,7 +80,7 @@ public void postProcessEnvironment(
           .getPropertySources()
           .addFirst(new MapPropertySource("CLOUD_SQL_DATA_SOURCE_URL", primaryMap));
 
-      setCredentials(sqlProperties, propertiesRetriever.getGcpProperties());
+      CredentialsPropertiesSetter.setCredentials(sqlProperties, propertiesRetriever.getGcpProperties(), LOGGER);
```

Comment:
No need to pass the logger; define a static logger in the `CredentialsPropertiesSetter` if it's needed. Per-class loggers are useful because end users can control logging verbosity per package/class if needed.